File: ejerciciosMariet/Java-ejercicios/Ejercicio_8.java
import java.util.Scanner;

public class Ejercicio_8 {
   public static void main(String[] args) {
      Scanner scanner = new Scanner(System.in);
      System.out.print("Ingrese el sueldo base del vendedor: ");
      double sueldoBase = scanner.nextDouble();
      System.out.print("Ingrese el valor de la primera venta: ");
      double venta1 = scanner.nextDouble();
      System.out.print("Ingrese el valor de la segunda venta: ");
      double venta2 = scanner.nextDouble();
      System.out.print("Ingrese el valor de la tercera venta: ");
      double venta3 = scanner.nextDouble();
      double comisionTotal = (venta1 + venta2 + venta3) * 0.1;
      double totalRecibido = sueldoBase + comisionTotal;
      System.out.println("El dinero a obtener por comisiones es: " + comisionTotal);
      System.out.println("El total que recibió en el mes es: " + totalRecibido);

      scanner.close();
   }
}

File: ejerciciosMariet/Python-ejercicios/ejercicio8.py
sueldoBase = float(input("Ingrese el sueldo base del vendedor: "))
venta1 = float(input("Ingrese el valor de la primera venta: "))
venta2 = float(input("Ingrese el valor de la segunda venta: "))
venta3 = float(input("Ingrese el valor de la tercera venta: "))

comisionTotal = (venta1 + venta2 + venta3) * 0.1
totalRecibido = sueldoBase + comisionTotal

print("El dinero a obtener por comisiones es:", comisionTotal)
print("El total que recibió en el mes es:", totalRecibido)